### DISCUSSION TITLE
关于“电犀牛 R68S 的安装方法”

### DISCUSSION THREAD

**Discussion by cherinyy** [ophub/amlogic-s9xxx-armbian#1438] [Rockchip] [answered]
https://github.com/ophub/amlogic-s9xxx-armbian/blob/60ea5de20f65b90cc8dc33d6eeffbe999920616f/build-armbian/documents/README.cn.md?plain=1#L279

这个“请先在高级功能里擦除”，指的是 `擦除所有` 这个选项吗？

![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/9050713/a949a760-d4fd-47ca-9f8c-f62c5e20a02a)

https://github.com/ophub/amlogic-s9xxx-armbian/blob/60ea5de20f65b90cc8dc33d6eeffbe999920616f/build-armbian/documents/README.cn.md?plain=1#L276-L278

另外这个步骤是不是过于繁琐了，我看从仓库里下载的 [RKDevTool](https://github.com/ophub/kernel/releases/download/tools/FastRhino_r68s_RKDevTool_Release_v2.86___DriverAssitant_v5.1.1.tar.gz) 实际上已经写好了选项，不需要再新建项吧？

![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/9050713/9ff8a189-cbb2-4280-aed9-c80d8d1c2cbf)

**Comment by ophub** [comment, marked as answer]
对了就直接使用即可。你理解的对。
